### DISCUSSION TITLE
Setting Default Column Values for Folders?

### DISCUSSION THREAD

**Discussion by nfritzen** [pnp/pnpcore#934] [Q&A] [answered]
I am trying to set default column values for folders as you can do it in the SharePoint UI via _Library Settings - Column Default Value Settings_ by chosing the required folder and defining a default value for the corresponding column.
It is also possible with the **[Set-PnPDefaultColumnValues](https://pnp.github.io/powershell/cmdlets/Set-PnPDefaultColumnValues.html)** cmdlet via PowerShell and defining the _-Folders_ parameter including the path of the folder.

Is there any way to make this work via PnP Core?

**Comment by jansenbe** [comment, marked as answer]
@nfritzen : this still needs to be added. I'll make an issue from this discussion to track the ask.

**Reply by jansenbe** [reply]
@nfritzen : this feature has been implemented and will appear in the next nightly build (version 1.7.50 or higher). Docs are still to be written, but usage is fairly simple and can be seen in the test case: https://github.com/pnp/pnpcore/blob/dev/src/sdk/PnP.Core.Test/SharePoint/ListTests.cs#L1975-L2048

**Reply by nfritzen** [reply]
Thank you very much!
